packages/oui-navbar/navbar.less

```diff
@@ -402,6 +402,9 @@
       font-size: @oui-navbar-menu-title-font-size;
       font-weight: @oui-navbar-menu-title-font-weight;
       color: @oui-navbar-menu-title-font-color;
+      overflow: hidden;
```

Comment:
We have a mixin for that: `#oui > .text-ellipsis();` :)

Here's the source: https://github.com/ovh-ux/ovh-ui-kit/blob/05dda64a54e1dd9a3dc120baeb446ebcb2051d90/packages/oui-typography/_mixins.less#L17

Reply:
@AxelPeter used the mixin instead, thanks :)